client/src/components/LogInModal.jsx

```diff
@@ -42,49 +38,77 @@ const LogInModal = ({ isOpen, onClose }) => {
                 const userStatus = await response.json();
 
                 //If there is a user with this username, log them in
-                if (userStatus.exists) {
-                    
+                if (userStatus.exists && userStatus.authorized) {
                     //Set the logged user state to the user id we will need to get the correct data
                     setLoggedUser({userId: userStatus.userId, name: userStatus.name});
+                    
+                    //Store access token locally to be used for authorization of all server calls
+                    localStorage.setItem("token", userStatus.accessToken);
```

Comment:
Something to implement in the future is to have a more unique name for the local storage key to avoid any conflicts with other apps that might also use `token`. 
Something my mentor mentioned was using a namespace for the data I wanted to store, (this [article](https://medium.com/@emadalam/namespace-localstorage-e2d1d2e68b20) talks about how to namespace your local storage) so instead of `token`, I used `MCC_Token` (My Crochet Corner - my app name) as my namespace to prepend to the access token.

Reply:
Woah that article was fascinating, thank you! I changed it to CaT_Token for Click and Treat and am so amused by the realization that the project initials are CaT. Thanks, Abbie! This change can be found in commit [11eac37](https://github.com/tati-guzman/click-and-treat/pull/8/commits/11eac370f7c4e474ec6a8b297493e0fe37a2b8bf)